### ISSUE TITLE
EPEL repo for CentOS 5 fails when using "yum --security check-update"

### ISSUE THREAD

**Issue by dalston** [fedora-infra/bodhi#380]
The command "yum --security check-update" fails and gives a return code of "1" as of Aug 23.  The repodata folder's timestamps show that it was modified on Aug 22 at about 13:53   (see mirror http://mirror.cs.pitt.edu/epel/5/x86_64/repodata/).

yum seems to behave normally when I do any one of these..
- I remove the "--security" option
- I disable the EPEL repo

This still happens after running "yum clean all"

Is the EPEL repo for CentOS 5 no longer compatible with the "--security" yum plugin?

**Comment by dalston** [comment]
Update:  It looks like the problem may be with accessing the mirror list..

```bash
centos5$ sudo yum clean metadata
Password:
Loaded plugins: downloadonly, fastestmirror, priorities, security
21 metadata files removed
7 sqlite files removed
0 metadata files removed


centos5$ sudo /usr/bin/yum --security check-update ; echo $?
Loaded plugins: downloadonly, fastestmirror, priorities, security
Loading mirror speeds from cached hostfile
Could not retrieve mirrorlist http://mirrors.fedoraproject.org/mirrorlist?repo=epel-5&arch=x86_64 error was
[Errno 12] Timeout: <urlopen error timed out>
Error: Cannot find a valid baseurl for repo: epel
```

**Comment by mkirank** [comment]
Is there a workaround for this

**Comment by dalston** [comment]
ping.. this is still an issue

**Comment by ralphbean** [comment]
I think this was fixed by the changes in commit 01ef021833fe8, but afaik we haven't been able to push that out to our servers yet because our infrastructure is frozen for the upcoming beta release of Fedora.

**Comment by ryanlerch** [comment]
Closing this one off, based on @ralphbean 's comment and the fact this one has been sitting around for a while with no updates either way.

Feel free to re-open if this is still an issue.
